### PR TITLE
[SRK-91] More detailed optimization #34

### PR DESCRIPTION
**Pull request by DanielRrr**

### CODE REVIEW

**Comment by vrom911** [comment]
Is this one finished, @DanielRrr ?

**Comment by DanielRrr** [comment]
@vrom911, yes. I forgot to erase WIP.

**Comment by ChShersh** [comment]
@DanielRrr CI is failing. Could you fix that?

File: serokell-util.cabal [hunk 1]
```diff
@@ -63,10 +60,11 @@ library
                      , fmt
                      , formatting >= 6.2.0 && < 6.3
                      , hashable >= 1.2.4.0
-                     , lens
```
Comment (ChShersh) [inline serokell-util.cabal]: :+1:

File: src/Serokell/Util/Lens.hs [hunk 2]
```diff
@@ -28,60 +28,36 @@ infix 4 %%=
...
     :: MonadReader s m
-    => L.LensLike' (L.Magnified (Reader s) a) s t
-    -> ReaderT t L.Identity a
+    => L.LensLike' (Mtl.Magnified (Reader s) a) s t
+    -> ReaderT t Identity a
     -> m a
-magnify' l = reader . runReader . L.magnify l
+magnify' l = reader . runReader . Mtl.magnify l
 
 -- | This isomorphism can be used to convert to or from an instance of 'IsList'.
```
Comment (ChShersh) [inline src/Serokell/Util/Lens.hs]: Would be great to remove documentation as well but whatever.

File: serokell-util.cabal [hunk 3]
```diff
@@ -63,10 +60,11 @@ library
...
                      , log-warper ^>= 1.8.9
+                     , microlens
+                     , microlens-mtl
+                     , microlens-platform
```
Comment (ChShersh) [inline serokell-util.cabal]: `microlens-platform` already includes other `microlens` packages so You don't need to add them to `.cabal` if you use `microlens-platform`. Other question: why you need `microlens-platform` at all? :thinking:

File: stack.yaml [hunk 4]
```diff
@@ -4,7 +4,5 @@ nix:
     packages: [openssh,git]
 
 extra-deps:
-- aeson-options-0.0.0
-- log-warper-1.8.9
+- o-clock-0.1.1
 - universum-1.1.0
```
Comment (vrom911) [inline stack.yaml]: we can use `lts-11.5` and then no need in these ones.

**Pull request merged**